project.clj

```diff
@@ -8,14 +10,15 @@
 
   :min-lein-version "2.7.1"
 
-  :parent-project {:coords [puppetlabs/clj-parent "5.2.9"]
+  :parent-project {:coords [puppetlabs/clj-parent "7.0.1"]
```

Comment:
```suggestion
  :parent-project {:coords [puppetlabs/clj-parent "7.2.7"]
```

Reply:
Should it be "8.0.0" actually once released?

Reply:
At this point updating to the latest is the safest.  I'm not sure clj-parent will be 8.0.0 as we aren't introducing breaking changes, but rather adding new dependencies.